runtime/src/main/java/io/quarkiverse/loggingjson/config/ConfigSocket.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkiverse.loggingjson.config;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.WithDefault;
+
+@ConfigGroup
+public interface ConfigSocket extends ConfigFormatter {
+    /**
+     * Determine whether to enable the JSON socket formatting extension, which disables "normal" socket formatting.
+     */
+    @WithDefault("true")
```

Comment:
I think it's best to make the default `false`, because maybe someone is using this extension and socket logging today.
This change can break a logging pipeline.

I also thinks, you need to enable socket logging in the  [test config](https://github.com/quarkiverse/quarkus-logging-json/blob/aa84511d9de63d42c1fce30411e90d0a6fbb1ce3/deployment/src/test/resources/application-json.properties#L7)